<launch>
  <!-- turtlebot3_teleop_key already has its own built in velocity smoother -->
  <!-- Following Second argument in 'args' can be 'wheel' or 'lidar'. It determines the way hengel_bot determines its odometry -->
  <node pkg="sandbot_valve_control" type="publisher.cpp" name="valve_input_publisher" output="screen">
  </node>
  <node pkg="sandbot_valve_control" type="valve_control.cpp" name="valve_control" output="screen">
  </node>
</launch>
